src/App.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom'
+import './App.css';
+import data from './data/resume.json';
+
+function BuildBasicsElement(basics) {
```

Comment:
it would be good to move each of these components into a separate file to make the project more readable and easier to follow. 

Then you would import each component at the top of this file